test/Microsoft.Azure.Devices.Edge.Test/SetupFixture.cs

```diff
@@ -19,15 +20,15 @@ public class SetupFixture
 
         public SetupFixture()
```

Comment:
For Windows IoT Core, it will force to reboot after install IoT Edge daemon.  After restarted, it will run e2e tests only without IoT Edge daemon installation.  Just FYI, we don't need to address this comment in this PR; but you may need to understand more about the test flow in Windows IoT Core.

Reply:
Yes, we'll definitely need to see how we make this work in the Windows IoT Core scenario.